provider/vsphere/internal/vsphereclient/createvm.go

```diff
@@ -202,11 +200,12 @@ func (c *Client) CreateTemplateVM(
 	// provisioners (i.e. each model's) run on the same controller
 	// machine, so taking a machine lock ensures that only one
 	// process is updating VMDKs at the same time. We lock around
-	// access to the series directory.
+	// access to the base directory.
 	// There is no need for a special case with bootstrapping as
 	// it only occurs once.
+	track := strings.ReplaceAll(ovaArgs.Base.Channel.Track, ".", "")
 	release, err := c.acquireMutex(mutex.Spec{
-		Name:  "vsphere-" + ovaArgs.Series,
+		Name:  fmt.Sprintf("vsphere-%s-%s", ovaArgs.Base.OS, track),
```

Comment:
The name of the template in the folder seems to still have a `.` in it, should this match that?

Reply:
This name here is just the name of a mutex we acquire, while reading the dir. So they can be different